cli/src/pcluster/config/cluster_config.py

```diff
@@ -3179,6 +3191,11 @@ def _register_validators(self, context: ValidatorContext = None):  # noqa: C901
                     ]
                     for validator in flexible_instance_types_validators:
                         self._register_validator(validator, **validator_args)
+                    self._register_validator(
```

Comment:
Why add the `EnableSingleAvailabilityZoneValidator` in a loop for each compute resource when we define the Parameters to be queue Specific and do not take any values from those said Compute resources?